RebbleOS/vibrate.c

```diff
@@ -23,18 +23,37 @@
 #include "vibrate.h"
 #include "task.h"
 #include "semphr.h"
+#include <stdbool.h>
 
+#define VIBRATE_QUEUE_MAX_WAIT_TICKS  20    // chosen by fair dice roll
 
-static uint16_t VIBRO_SHORT[] = { 255, 1000 };
-static uint16_t VIBRO_TAP_TAP[] = { 255, 750, 0, 750, 255, 750 };
+
+/*
+ * Initialization of default patterns.
+ * 
+ * TODO Maybe load these dinamically at some point
+ */
```

Comment:
Maybe add a description of what each of these patterns achieves. We seem to have lost context a little